bip-XXXX.mediawiki

```diff
@@ -0,0 +1,449 @@
+<pre>
+  BIP: ???
+  Layer: Consensus (soft / hard fork)
+  Title: Logarithm of transaction fee limits block size. Game theoretic scaling on chain.
+  Author: Simon Cavegn <random@gmx.ch>
+  Comments-Summary: None
+  Comments-URI: https://github.com/bitcoin/bips/wiki/Comments:BIP-XXXX
+  Status: Draft
+  Type: Standards Track
+  Created: 2019-03-28
+</pre>
+
+==Abstract==
+
+Logarithm of transaction fee limits block size.
+
+If a sender is willing to pay a large fee, then a transaction may be allowed in a block even if blocksize for lower fee-transactions is full. Yet the blocksize is limited harshly by a formula.
+
+==Motivation==
+
+# Keep block space small.
+# Waste less with spam transactions.
+# Auto balance Fees: Increase very low fees, Descrease very high fees.
+# Allow larger size when sender pays a lot.
+# Allow wallets to calculate/display how much average free block space there is for each fee price bucket.
+# Allow senders to have more control about how the fee/priority of their transaction will behave, especially in the case of increased adoption in the future.
```

Comment:
The motivation falls short in describing what inadequacies of the existing protocol are being addressed by this BIP.